.github/workflows/create-js-release.yaml

```diff
@@ -19,53 +19,102 @@ jobs:
 
           - host: macos-latest
             target: 'x86_64-apple-darwin'
+            architecture: x64
             build: |
-              npm run build
+              yarn build
+              strip -x polars/*.node
 
           - host: macos-latest
             target: "aarch64-apple-darwin"
+            architecture: x64
             build: |
               sudo rm -Rf /Library/Developer/CommandLineTools/SDKs/*;
               export CC=$(xcrun -f clang);
               export CXX=$(xcrun -f clang++);
               SYSROOT=$(xcrun --sdk macosx --show-sdk-path);
               export CFLAGS="-isysroot $SYSROOT -isystem $SYSROOT";
-              npm run build -- --target aarch64-apple-darwin
+              yarn build --target aarch64-apple-darwin
+              strip -x polars/*.node
 
             # Windows
             ##########
 
           - host: windows-latest
+            architecture: x64
             target: "x86_64-pc-windows-msvc"
-            build: npm run build
+            build: yarn build
 
           - host: windows-latest
             target: "i686-pc-windows-msvc"
-            setup: |
-              choco install nodejs-lts --x86 -y --force
-              echo "C:\\Program Files (x86)\\nodejs" >> $GITHUB_PATH
+            architecture: x86
             build: |
-              export CARGO_PROFILE_RELEASE_CODEGEN_UNITS=32;
+              export CARGO_PROFILE_RELEASE_CODEGEN_UNITS=128;
```

Comment:
If we set this to 1 we get a faster binary. That's definitely worth it.

Reply:
Let's also set LTO=true

Reply:
There are codegen issues in Rust while compiling i686-windows-msvc target： https://github.com/rust-lang/rust/issues/67497.
So I disable it only in target `i686-pc-windows-msvc`

Reply:
Right, maybe we could add a comment to that PR so that we remember why we did it.